charts/trustify/templates/helpers/_infrastructure.tpl

```diff
@@ -44,6 +44,8 @@ Arguments (dict):
   value: parentbased_traceidratio
 - name: OTEL_TRACES_SAMPLER_ARG
   value: "0.1"
+- name: OTEL_EXPORTER_OTLP_ENDPOINT
+  value: "http://infrastructure-otelcol:4317"
```

Comment:
That hardcodes the value. Which might not work for all examples. As this is not the "example infrastructure" chart, but the one we use to deploy our application, we should add a bit more flexibility here.

I think it might make sense adding a property to maybe the `tracing` object for this.

Reply:
Ok I'll work on that thanks for the reivew :+1: